image/templates/helm/stackrox-secured-cluster/internal/config-shape.yaml

```
@@ -129,6 +129,8 @@ collector:
   priorityClassName: null # string
   disableSELinuxOptions: null # bool
   seLinuxOptionsType: null # string
+  containerdEnable: true # bool
+  crioEnable: true # bool
```

consider: allow extending the respective options by using an object. 

```suggestion
  containerd:
    enabled: true #bool
  crio:
    enabled: true #bool
    # Examples of options that could be added in the future
    # socketPath: /run/crio/crio.sock
    # mountPath: ...
```